### PR TITLE
Disable Network Permission Test Due to Test Instability 

### PR DESCRIPTION
**Pull request by peterschwarz**
Signed-off-by: Peter Schwarz <pschwarz@bitwise.io>

### CODE REVIEW

File: bin/run_tests [hunk 1]
```diff
@@ -341,8 +341,8 @@ test_integration() {
     copy_coverage .coverage.test_basic_auth_proxy
     run_docker_test test_transactor_permissioning
     copy_coverage .coverage.test_transactor_permissioning
-    run_docker_test test_network_permissioning --timeout 800
-    copy_coverage .coverage.test_network_permissioning
+    # run_docker_test test_network_permissioning --timeout 800
```
Comment (jsmitchell) [inline bin/run_tests]: Don't commit commented out code

File: integration/sawtooth_integration/tests/test_network_permissioning.py [hunk 2]
```diff
@@ -544,7 +544,8 @@ def validator_genesis_init(sawtooth_home_genesis,
     subprocess.run([
         'sawset', 'proposal', 'create',
         '-k', priv,
-        'sawtooth.identity.allowed_keys={},{}'.format(identity_pub_key, priv_key),
+        'sawtooth.identity.allowed_keys={},{}'.format(
```
Comment (jsmitchell) [inline integration/sawtooth_integration/tests/test_network_permissioning.py]: Is this a lint change?

**Comment by agunde406** [comment]
Can you create a jira story to investigate this further so it doesn't get forgotten?

**Comment by peterschwarz** [comment]
Created [STL-906](https://jira.hyperledger.org/browse/STL-906)

**Comment by peterschwarz** [comment]
Closing in favor of #1188, which cherry-picked the commit

**Pull request closed without merging**